Shopify/polaris-react
# Update storybook to 6.1

### WHY are these changes introduced?

Keeping up to date

Storybook 61 comes with a revamed sidebar (aka the search no longer terrible) and faster boot up times

### WHAT is this pull request doing?

Update storybook to v6.1


### To Test

Check out locally and click around.

See that live reloading of README and components work
